modules/mod_ginger_base/controllers/controller_rest_resources.erl

```diff
@@ -103,8 +103,9 @@ to_json(Req, State = #state{mode = collection}) ->
                    wrq:req_qs(Req)
                   )
                 ),
+        EdgePath = parse_path(proplists:get_value("edge_path", wrq:req_qs(Req), "")),
```

Comment:
Does this mean one can only traverse/include a single edge path in a single resource request? I would think we need to be able to specify multiple paths.

Reply:
Currently we gain two abilities with this new query parameter:
- Specify depth of edges
- Specify a filter for the edges at each depth
Together I called this an 'edge path'.

But indeed, we could want to resolve multiple 'edge paths'. I think naively, we could solve this by just taking a list of 'edge paths' and running `with_deep_edges` for each of the paths, but of course we would be wasting effort by traveling the same edges twice.

Reply:
Yes, maybe good to start out with this and if we see we need more we can easily extend.